boxoffice/extapi/stats.py

```diff
@@ -0,0 +1,37 @@
+from boxoffice import db, app
+from boxoffice.models import ItemCollection, LineItem, Order, ORDER_STATUS
+from datetime import datetime as dt
+import requests
+from urllib import urlencode
+import json
+from tabulate import tabulate
+
+def query_for_ItemCollectionIds():
```

Comment:
We use underscore case as a naming convention in Python. So, `ItemCollectionIds` would be `item_collection_ids`.